cmd/skaffold/app/cmd/delete.go

```diff
@@ -43,7 +52,34 @@ func NewCmdDelete() *cobra.Command {
 }
 
 func doDelete(ctx context.Context, out io.Writer) error {
-	return withRunner(ctx, out, func(r runner.Runner, _ []util.VersionedConfig) error {
-		return r.Cleanup(ctx, out, dryRun)
+	return withRunner(ctx, out, func(r runner.Runner, configs []util.VersionedConfig) error {
+		workDir, err := sutil.RealWorkDir()
```

Comment:
can we move the added code here into a function to reuse here and in `cmd/dev.go`? Looks like it's almost the same code

Reply:
Yep, moved deduped the logic to a method in cmd/util.go and reuse it in `cmd/dev.go` and `cmd/delete.go`